Scrumdinger/Views/DetailEditView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  DetailEditView.swift
+//  Scrumdinger
+//
+//  Created by Life Boss on 7/16/23.
+//
+
+import SwiftUI
+
+struct DetailEditView: View {
+    @State private var scrum = DailyScrum.emptyScrum
+    @State private var newAttendeeName = ""
+    
+    var body: some View {
+        Form {
+            Section (header: Text("Meeting Info")) {
+                TextField("Title", text: $scrum.title)
```

Comment:
☝️ Note the distinction of using `$scrum` vs `scrum`

Appending an initial `$` indicates that you are passing a **binding** to the property value to contexts where you need read-write access to the value:

https://github.com/meggra7/scrumdinger/blob/32223ef741e477a5e7bfe02df2463b0c69045139/Scrumdinger/Views/DetailEditView.swift#L18

You can and should omit the initial `$` once within the context that has reference to the bound property. You can then directly mutate the property as needed. Or, if the context was passed a non-bound version of the property, you will only be able to read but not write values.

https://github.com/meggra7/scrumdinger/blob/32223ef741e477a5e7bfe02df2463b0c69045139/Scrumdinger/Views/DetailEditView.swift#L22

_See [State](https://developer.apple.com/documentation/swiftui/state)._